fenix/app/src/main/res/values/strings.xml

```diff
@@ -330,6 +330,17 @@
     <string name="juno_onboarding_enable_notifications_positive_button" tools:ignore="UnusedResources">Turn on notifications</string>
     <!-- Text for the button dismiss the screen and move on with the flow -->
     <string name="juno_onboarding_enable_notifications_negative_button" tools:ignore="UnusedResources">Not now</string>
+    <!-- Title for add search widget screen.
+        Note: The word "Firefox" should NOT be translated -->
+    <string name="juno_onboarding_add_search_widget_title" tools:ignore="UnusedResources">Try the Firefox search widget</string>
```

Comment:
Should we be substituting the `Firefox` string here with the app_name?

Reply:
This might be a *blocking change* depending on the details surrounding the string for this project.

Reply:
> Should we be substituting the `Firefox` string here with the app_name?

I am not sure about this tbh. There is a similar situation with the enable notification screen. Would be ideal to get someone from l10n on this, @gabrielluong @dnarcese do you know someone I can ping?

Reply:
@flodolo @Delphine Could you provide any guidance regarding these strings here?

Reply:
I believe these strings will be need to be added to the exception similar to https://github.com/mozilla-l10n/android-l10n/pull/646#issuecomment-1526465800. These are onboarding strings and therefore, I believe we wanted the app_name to not be substituted in this case.

Reply:
We can add exceptions on our side (linter). The goal of not hard-coding the name is to make things easier for someone repackaging Firefox, but I doubt experiments would be relevant for that group.